.github/workflows/cd.yml

```diff
@@ -160,6 +131,53 @@ jobs:
         run: |
           npx lerna publish from-package --yes
 
+      - name: Install Mustache
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') }}
+        run: curl -sSL https://git.io/get-mo -o mo && chmod +x mo && sudo mv mo /usr/local/bin/
+
+      - name: generate templates
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') }}
+        run: .github/scripts/template-zip-autogen.sh
+
+      - name: release templates' prerelease version to github
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && github.event.inputs.preid != 'stable' }}
+        uses: marvinpinto/action-automatic-releases@latest
+        with:
+          repo_token: ${{ secrets.CD_PAT }}
+          prerelease: true
+          automatic_release_tag: "templates@0.0.0-${{github.event.inputs.preid}}"
+          files: |
+            ./*.zip
+
+      - name: Create Templates Stable Release
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && github.event.inputs.preid == 'stable' }}
+        uses: ncipollo/release-action@v1.7.3
+        with:
+          artifacts: ./*.zip
+          name: 'Release for $(tr ' ' '\n' <<< '${{ steps.version-change.outputs.CHANGED }}' | grep 'templates@')'
+          token: ${{ secrets.github_token }}
+          tag: $(tr ' ' '\n' <<< '${{ steps.version-change.outputs.CHANGED }}' | grep 'templates@')
+          allowUpdates: true
+
+      - name: Generate Tag List
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && github.event.inputs.preid == 'stable' }}
+        run: |
+          tr ' ' '\n' <<< '${{ steps.version-change.outputs.CHANGED }}' | grep 'templates@' > ./template-tags.txt
+
+      - name: Update Template Tag list Release
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && github.event.inputs.preid == 'stable' }}
```

Comment:
please update the tag list even it is not stable, in case we have other prerelease suffix.

Reply:
but this action only release stable template can trigger it

Reply:
Yes, please help to remove the limits. Because it is not hard-coded with rc/alpha anymore, it is 0.0.0-{github.event.inputs.preid}, so we'd better to consider the extension.

Reply:
branch model only allows alpha, rc, stable release, others is forbidden